pom.xml

```diff
@@ -126,66 +138,88 @@
       <dependency>
         <groupId>org.codehaus.sonar</groupId>
         <artifactId>sonar-deprecated</artifactId>
-        <version>${sonar.version}</version>
+        <version>${version.sonar}</version>
       </dependency>
       <dependency>
         <groupId>org.codehaus.sonar</groupId>
         <artifactId>sonar-testing-harness</artifactId>
-        <version>${sonar.version}</version>
+        <version>${version.sonar}</version>
       </dependency>
       <dependency>
         <groupId>org.sonarsource.sslr</groupId>
         <artifactId>sslr-core</artifactId>
-        <version>${sslr.version}</version>
+        <version>${version.sslr}</version>
       </dependency>
       <dependency>
         <groupId>org.sonarsource.sslr</groupId>
         <artifactId>sslr-toolkit</artifactId>
-        <version>${sslr.version}</version>
+        <version>${version.sslr}</version>
       </dependency>
       <dependency>
         <groupId>org.sonarsource.sslr-squid-bridge</groupId>
         <artifactId>sslr-squid-bridge</artifactId>
-        <version>2.6.1</version>
+        <version>${version.sslr-squid-bridge}</version>
       </dependency>
       <dependency>
         <groupId>org.sonarsource.sslr</groupId>
         <artifactId>sslr-testing-harness</artifactId>
-        <version>${sslr.version}</version>
+        <version>${version.sslr}</version>
       </dependency>
       <dependency>
         <groupId>junit</groupId>
         <artifactId>junit</artifactId>
-        <version>4.10</version>
+        <version>${version.junit}</version>
       </dependency>
       <dependency>
         <groupId>org.mockito</groupId>
         <artifactId>mockito-all</artifactId>
-        <version>1.9.0</version>
+        <version>${version.mockito}</version>
       </dependency>
       <dependency>
         <groupId>org.easytesting</groupId>
         <artifactId>fest-assert</artifactId>
-        <version>1.4</version>
+        <version>${version.fest-assert}</version>
       </dependency>
       <dependency>
         <groupId>ch.qos.logback</groupId>
         <artifactId>logback-classic</artifactId>
-        <version>0.9.30</version>
+        <version>${version.logback}</version>
       </dependency>
       <dependency>
         <groupId>org.slf4j</groupId>
         <artifactId>slf4j-api</artifactId>
-        <version>1.6.2</version>
+        <version>${version.slf4j}</version>
       </dependency>
       <dependency>
         <groupId>com.thoughtworks.xstream</groupId>
         <artifactId>xstream</artifactId>
-        <version>1.3.1</version>
+        <version>${version.xstream}</version>
+      </dependency>
+      <dependency>
+        <groupId>org.apache.maven</groupId>
+        <artifactId>maven-project</artifactId>
```

Comment:
@pynicolas Yes, probably. My plan was to check all dependencies and update the oldest ones in a post step to the update to LTS.